addon/components/sortable-group.js

```diff
@@ -18,29 +19,35 @@ export default Component.extend({
     @property items
     @type Ember.NativeArray
   */
-  items: computed(() => { return a(); }),
+  items: computed({
+    get() { return a(); }
```

Comment:
**Maybe** the deprecation warning had something to do with calling `set` on computed property with no explicit set defined.